src/freva/cli/esgf.py

```diff
@@ -75,7 +76,7 @@ def __init__(
             "--query",
             default=None,
             type=str,
-            help=("Display results from <list> queried fields"),
+            help=("Query fields from ESGF and group them per dataset"),
         )
         self.parser.add_argument(
```

Comment:
Could you please add a `--json` flag, that prints any output as a json string. Users can then use jq to parse the output rather than using awk or what not. I've done that in the jupyter improvements branch for the plugin command.

Reply:
I will look into your jupyter branch for that.

but I do not know whether for esgf is particularly necessary? as all the query options except for the wget script retrieve metadata and filepaths based on URLs (gdap as well). @antarcticrainforest ?

Reply:
I was taking a look at the freva.databrowser but I did not see anything related to json dumps in the methods.

depending on whether we call 

Could you give me an example? Right now the esgf methods are separated similar to how you did with databrowser:

- esgf_browser returns a list of strings,
- esgf_facets:  dict[str , list[str]]
- esgf_datasets: list[tuple[str, str]]
- esgf_download: Union[str, Path],    wget_script to download the files


I will modify the names to not make it esgf.esg_browser but esgf.browser etc.
and I will try to put up to date my branch with all the other changes of main branch on the mean time

Reply:
The `darabrowser` cli doesn't have that yet. I meant more the history or plugin cli

Reply:
I will see on how to do it later on another branch, but I would like to for once add the documentation of the esgf, that is at this moment inexistent. and hence, that method undocumented and officially unavailable. otherwise I will never finish this MR.